admin_guide/scheduling/scheduler.adoc

```diff
@@ -269,54 +269,57 @@ These predicates do not take any configuration parameters or inputs from the
 user. These are specified in the scheduler configuration using their exact
 name.
 
+[[default-predicates]]
+==== Default Predicates
+
+The default scheduler policy includes the following predicates:
+
 **_NoVolumeZoneConflict_** checks that the volumes a pod requests
-are available in the zone. _The default scheduler policy includes this predicate._
+are available in the zone. 
 ----
 {"name" : "NoVolumeZoneConflict"}
 ----
 
 **_MaxEBSVolumeCount_** checks the maximum number of volumes that can be attached to an AWS instance.
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "MaxEBSVolumeCount"}
 ----
 
 **_MaxGCEPDVolumeCount_** checks the maximum number of Google Compute Engine (GCE) Persistent Disks (PD).
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "MaxGCEPDVolumeCount"}
 ----
 
 **_MatchInterPodAffinity_** checks if the pod affinity/antiaffinity rules permit the pod.
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "MatchInterPodAffinity"}
 ----
 
 **_NoDiskConflict_** checks if the volume requested by a pod is available.
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "NoDiskConflict"}
 ----
 
 **_PodToleratesNodeTaints_** checks if a pod can tolerate the node taints.
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "PodToleratesNodeTaints"}
 ----
 
 **_CheckNodeMemoryPressure_** checks if a pod can be scheduled on a node with a memory pressure condition.
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "CheckNodeMemoryPressure"}
 ----
 
 **_CheckNodeDiskPressure_** checks if a pod can be scheduled on a node with a disk pressure condition.
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "CheckNodeDiskPressure"}
 ----
 
+[other-predicates]]
```

Comment:
@mburke5678 - this doesn't look correct. Can you check in other branches as well?

Reply:
@vikram-redhat Will do.

Reply:
Fixed https://github.com/openshift/openshift-docs/pull/8219